src/wp-includes/block-template.php

```diff
@@ -5,6 +5,31 @@
  * @package WordPress
  */
 
+/**
+ * Adds necessary filters to use 'wp_template' posts instead of theme template files.
+ *
+ * @since 5.9.0
+ */
+function add_template_loader_filters() {
```

Comment:
Are we okay with the naming here? `template_loader` has a bit of a ring of the filename where this logic is located in Gutenberg: https://github.com/WordPress/gutenberg/blob/8eb1f341285362f6a131fcba81a8d054b6b5b5c5/lib/full-site-editing/template-loader.php#L11, but it might make less sense here 🤔

Reply:
Is it expected that third party developers should call this? I assume "no". If that's the case then I'd prefix the name with `_` or `_wp_` and add `@access private` to the doc block.